proxy/interceptor.go

```diff
@@ -2884,6 +2939,64 @@ func visitFailures(ctx *VisitFailuresContext, options *VisitFailuresOptions, obj
 				return err
 			}
 
+		case *command.ScheduleNexusOperationCommandAttributes:
+			if o == nil {
+				continue
+			}
+			ctx.Parent = o
+			if err := visitFailures(
+				ctx,
+				options,
+			); err != nil {
+				return err
+			}
+
+		case *errordetails.MultiOperationExecutionFailure:
+			if o == nil {
+				continue
+			}
+			ctx.Parent = o
+			if err := visitFailures(
+				ctx,
+				options,
+				o.GetStatuses(),
+			); err != nil {
+				return err
+			}
+
+		case []*errordetails.MultiOperationExecutionFailure_OperationStatus:
+			for _, x := range o {
+				if err := visitFailures(ctx, options, x); err != nil {
+					return err
+				}
+			}
+
+		case *errordetails.MultiOperationExecutionFailure_OperationStatus:
+			if o == nil {
+				continue
+			}
+			ctx.Parent = o
+			if err := visitFailures(
+				ctx,
+				options,
+				o.GetDetails(),
+			); err != nil {
+				return err
+			}
+
+		case *errordetails.QueryFailedFailure:
```

Comment:
I think this one we actually do need to handle because their is a payload here?

Reply:
Yes, we need to handle that as part of `visitPayloads` (`visitFailures` here is a separate concern)